source/shared/core_conn.cpp

```diff
@@ -321,13 +321,12 @@ bool core_compare_error_state( _In_ sqlsrv_conn* conn,  _In_ SQLRETURN rc, _In_
 
 bool core_search_odbc_driver_unix( _In_ DRIVER_VERSION driver_version )
 {
+#ifndef _WIN32
     char szBuf[DEFAULT_CONN_STR_LEN+1];     // use a large enough buffer size
     WORD cbBufMax = DEFAULT_CONN_STR_LEN; 
     WORD cbBufOut;
     char *pszBuf = szBuf;
-    bool found_driver = false;
 
-#ifndef _WIN32
     // get all the names of the installed drivers delimited by null characters
```

Comment:
why removed `#ifndef`?

Reply:
it's not removed. I moved it up to line 324. Those variables are not needed in Windows.